packages/subcommands/src/lib/Subcommand.ts

```diff
@@ -249,14 +322,54 @@ export class Subcommand<PreParseReturn extends Args = Args, O extends Subcommand
 		});
 	}
 
-	async #handleMessageRun(message: Message, args: Args, context: MessageCommand.RunContext, subcommand: SubcommandMappingMethod) {
+	async #getMessageParametersAsString(args: Args): Promise<Partial<Pick<MessageCommandDeniedPayload, 'parameters'>>> {
+		args.save();
+		const parameters = await args.restResult('string');
+		args.restore();
+
+		const params: Partial<Pick<MessageCommandDeniedPayload, 'parameters'>> = {};
+
+		if (parameters.isOk()) {
+			params.parameters = parameters.unwrap();
+		}
+
+		return params;
+	}
```

Comment:
What exactly is the point of this?

Reply:
Mimicking the payload that normal preconditions get

See https://github.com/sapphiredev/framework/blob/9103253c8da170c229dd87f5e22ef0031d96c436/src/optional-listeners/message-command-listeners/CorePreMessageCommandRun.ts#L9-L17

Which gets called from: https://github.com/sapphiredev/framework/blob/9103253c8da170c229dd87f5e22ef0031d96c436/src/optional-listeners/message-command-listeners/CorePrefixedMessage.ts#L40-L46

And it is also in the interface: https://github.com/sapphiredev/framework/blob/9103253c8da170c229dd87f5e22ef0031d96c436/src/lib/types/Events.ts#L455-L458